QUIZ APP/quiz.js

```diff
@@ -0,0 +1,228 @@
+document.addEventListener("DOMContentLoaded", function() {
+    
+    const questions = [
```

Comment:
Usually it's a good idea to move hard-coded data like this to a separate file, to separate data from the logic.